orthanc/orthanc-anon/config/orthanc.json

```diff
@@ -1,4 +1,29 @@
 {
+  "Dictionary": {
+    "000b,1001": ["LO", "UCLHPIXLProjectName", 1, 1, "UCLH PIXL"]
+  },
```

Comment:
Surprisingly to me, removing this section seems to not affect the outcome - is this because we're never actually modifying this value though the API, it's just being allowed through unmodified during the anonymisation?
Should it be left in regardless?

Reply:
It makes the display in the UI a bit nicer but other than that serves no purpose in orthanc-anon.

Reply:
That seems like a reason to keep it?

Reply:
Yeah I'm fine with that